loan-payment-skip/loan-payment-skip-json-contract.md

```diff
@@ -82,21 +82,20 @@ Any and all errors should be conveyed via the following structure:
 ```
 ### Response Messages & Corresponding Logging Error Messages
 
-*See Logging Error section for additional details
-
-| Client Error Message                                                                 | Logging Error Message                                                       | Notes |
-| ------------------------------------------------------------------------------------- |------------------------------------------------------------------------------------- |-----------------------------------------------|
-| We're sorry we've encountered an error. Please contact your credit union. | No loan types found in CFG Letter file | This error represents issues with the configuration letter file setup |
-|| Invalid loan tracking type: 1234 ||
-|| Invalid sub source code: 123 ||
-|| Invalid other action: 123 | Configuration error when other action is not 0 share fee comment or 1 loan fee comment |
-|| Test mode on but no test account(s) defined ||
-|| Invalid start/end date(s) ||
-|| Error Opening Letterfile [config file name]: [system generated letter file read error message] ||
-| We're sorry we've encountered an error. Please contact your credit union. | Invalid loan id: [loan id] | This type of error is a non-configuration file error |
-|| Attempt to skip a payment on an ineligible loan: [loan id] ||
-|| Invalid fee share id: [share id] ||
-|| Loan Record Update Failed. Error: [file maintenance system error message] ||
-|| Tracking Record Create Failed. Error: [file maintenance system error message] ||
-|| Skip Pay Fee Post Failed. Error=[transaction posting system error message] ||
+*All Errors beging with 'We're sorry we've encountered an error. Please contact your credit union.' followed by a Logging Error Message.
 
+| Logging Error Message                                                                | Additional Notes As Needed |
+|------------------------------------------------------------------------------------- |-----------------------------------------------|
+| No loan types found in CFG Letter file | This set of errors represents issues with the configuration letter file setup |
+| Invalid loan tracking type: 1234 ||
+| Invalid sub source code: 123 ||
+| Invalid other action: 123 | Configuration error when OA(other action) CFG file setting is not 0 (share fee comment) or 1 (loan fee comment) |
```

Comment:
There are some areas with info in parens with a space before the paren and some without the space....

'TNC(test newest' vs. '1 (loan '

pick one for consistency - my pref FWIW would be to have the space.

Reply:
Added a space before '(' after the CFG tag where needed